doc/devel/coding_guide.rst

```diff
@@ -1,103 +1,31 @@
-.. _coding-guide:
+.. _reviewers-guide:
 
-************
-Coding guide
-************
+********************
+Reviewers guideline
+********************
 
 .. _pull-request-checklist:
 
 Pull request checklist
 ======================
 
-This checklist should be consulted when creating pull requests to make
-sure they are complete before merging.  These are not intended to be
-rigidly followed---it's just an attempt to list in one place all of
-the items that are necessary for a good pull request.  Of course, some
-items will not always apply.
-
 Branch selection
 ----------------
 
 * In general, simple bugfixes that are unlikely to introduce new bugs
   of their own should be merged onto the maintenance branch.  New
   features, or anything that changes the API, should be made against
-  master.  The rules are fuzzy here -- when in doubt, try to get some
-  consensus.
-
-  * Once changes are merged into the maintenance branch, they should
-    be merged into master.
-
-Style
------
-
-* Formatting should follow `PEP8
-  <http://www.python.org/dev/peps/pep-0008/>`_.  Exceptions to these
-  rules are acceptable if it makes the code objectively more readable.
-
-  - You should consider installing/enabling automatic PEP8 checking in your
-    editor.  Part of the test suite is checking PEP8 compliance, things
-    go smoother if the code is mostly PEP8 compliant to begin with.
-
-* No tabs (only spaces).  No trailing whitespace.
+  master.  The rules are fuzzy here -- when in doubt, target master.
 
-  - Configuring your editor to remove these things upon saving will
-    save a lot of trouble.
-
-* Import the following modules using the standard scipy conventions::
-
-    import numpy as np
-    import numpy.ma as ma
-    import matplotlib as mpl
-    from matplotlib import pyplot as plt
-    import matplotlib.cbook as cbook
-    import matplotlib.collections as mcol
-    import matplotlib.patches as mpatches
-
-* See below for additional points about
-  :ref:`keyword-argument-processing`, if code in your pull request
-  does that.
-
-* Adding a new pyplot function involves generating code.  See
-  :ref:`new-pyplot-function` for more information.
+* Once changes are merged into the maintenance branch, they should
+  be merged into master.
 
 Documentation
 -------------
 
 * Every new feature should be documented.  If it's a new module, don't
   forget to add a new rst file to the API docs.
 
-* Docstrings should be in `numpydoc format
-  <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`_.
-  Don't be thrown off by the fact that many of the existing docstrings
-  are not in that format;  we are working to standardize on
-  `numpydoc`.
-
-  Docstrings should look like (at a minimum)::
-
-        def foo(bar, baz=None):
```

Comment:
I think keeping a minimal example around is a good idea.  Most people are doing to do this by pattern matching not by reading the spec :smiling_imp: .